terraform/addons/byo-firehose-logging-destination/firehose/version.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.52.0"
+      version = ">= 4.53.0"
```

Comment:
This shouldn't change

Reply:
@rfairburn Thanks for catching. 

@georgekarrv Gap in our release script. If there happens to be other versions in the file that match the current Fleet version, they also get replaced.

Reply:
Yes for each one I had been adding file exclusions but same file does make that tricky. Can update next round

Reply:
@georgekarrv Actually these aren't same files so maybe just a new file to add to the exclusions.